frontend/src/components/Header.tsx

```diff
@@ -0,0 +1,88 @@
+import HeaderDesktop from './HeaderDesktop';
+import HeaderMobile from './HeaderMobile';
+import { useEffect, useState} from 'react';
+import { FaMoon, FaSun } from "react-icons/fa";
+import LinksMenu from './LinksMenu';
+
+interface theme {
+    setDarkTheme: void;
+    darkTheme: boolean;
+};
+
+const Header = ({setDarkTheme, darkTheme}: theme) => {
+    //Array with objects content linksMenu
+    const [isDesktop, setIsDestkop] = useState(window.innerWidth > 760);
+    const [pathActive, setPathActive] = useState(window.location.hash.replace(`#`, ``));
+
+    const linksMenu = LinksMenu;
+
+    useEffect(() => {
+        const updateWindowDimensions = () => {
+            setIsDestkop(window.innerWidth > 855);
+        };
+        window.addEventListener("resize", updateWindowDimensions);
+    }, [window.innerWidth]);
+
+    return (
+        <>
+            {!isDesktop && <HeaderMobile 
+            pathActive={pathActive}
+            setPathActive={setPathActive}
+            linksMenu={linksMenu}
+            MenuLink={MenuLink}/>}
+            <header>
+                { isDesktop ? 
+                <HeaderDesktop 
+                pathActive={pathActive}
+                setPathActive={setPathActive}
+                linksMenu={linksMenu} 
+                MenuLink={MenuLink}/> : 
```

Comment:
Alteração: manter um padrão de case para nome de propriedades, neste caso, geralmente se usa camelCase:

```suggestion
                pathActive={pathActive}
                setPathActive={setPathActive}
                linksMenu={linksMenu} 
                menuLink={MenuLink}/> : 
```